src/app/bar-chart/bar-chart.component.ts

```diff
@@ -0,0 +1,79 @@
+import {Component, OnInit} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {Observable} from 'rxjs';
+
+@Component({
+  selector: 'app-bar-chart',
+  templateUrl: './bar-chart.component.html',
+  styleUrls: ['./bar-chart.component.css']
+})
+export class BarChartComponent implements OnInit {
+  urlMoodAvgGet = 'http://www.mocky.io/v2/5de5081a2e00004c0031fad0';
```

Comment:
please add comment that this will be replaced asap when the desired URL is available